src/components/posts/buttons/DeleteButton.js

```diff
@@ -0,0 +1,16 @@
+function DeleteButton(props) {
+  function deletePost() {
+    fetch(
+      `https://friendface-react-e85cf-default-rtdb.firebaseio.com/posts/${props.id}.json`,
+      {
+        method: "DELETE",
+      }
+    ).then(() => {
+      props.setRefreshPost(true);
```

Comment:
Although passing the `setRefreshPost` through the props is a lot of work (and there might be a better way) now that you have it there adding in the refreshing to other places is relatively straight forward

Reply:
yeah I definitely noticed that, although just had to catch myself after having used it too many times, when i didn't actually need to